docs/profiles.md

```diff
@@ -210,14 +210,16 @@ The following attributes can be read and set on the result in `process_turn`:
 
 Attribute          | Read/write? | Type    | Notes
 -------------------|-------------|---------|------------------------------------------------------
-direction_modifier | Read        | Enum    | Geometry of turn. Defined in `include/extractor/guidance/turn_instruction.hpp`
-turn_type          | Read        | Enum    | Priority of turn. Defined in `include/extractor/guidance/turn_instruction.hpp`
+angle              | Read        | Float   | Angle of turn in degrees (`0-360`: `0`=u-turn, `180`=straight on)
+number_of_roads    | Read        | Integer | Number of ways at the intersection of the turn
+is_u_turn          | Read        | Boolean | Is the turn a u-turn?
 has_traffic_light  | Read        | Boolean | Is a traffic light present at this turn?
 source_restricted  | Read        | Boolean | Is it from a restricted access road? (See definition in `process_way`)
 target_restricted  | Read        | Boolean | Is it to a restricted access road? (See definition in `process_way`)
-angle              | Read        | Float   | Angle of turn in degrees (`0-360`: `0`=u-turn, `180`=straight on)
-duration           | Read/write  | Float   | Penalty to be applied for this turn (duration in deciseconds)
 weight             | Read/write  | Float   | Penalty to be applied for this turn (routing weight)
+duration           | Read/write  | Float   | Penalty to be applied for this turn (duration in deciseconds)
+source_mode        | Read/write  | Enum    | Travel mode before the turn. Defined in `include/extractor/travel_mode.hpp`
```

Comment:
`source_mode` and `target_mode` are `Read/write` properties but not used after `process_turn` calls and can be `Read-only` properties. I guess `const` qualifiers are missing at https://github.com/Project-OSRM/osrm-backend/blob/6d801e7086cf7c12384611bda4f441ac3ce4dc17/include/extractor/extraction_turn.hpp#L42-L44

Reply:
👍